RELEASENOTES-1.4.docu

```diff
@@ -558,4 +558,14 @@
       in multiple inheritance: it allows any method that overrides multiple
       implementations from unrelated templates to individually reference and
       call the implementations it overrides.</add-note></build-id>
+  <build-id _6="next" _7="next"><add-note>Found a bug where DMLC did
+      not report errors when a non-existing template is instantiated within
+      an <tt>#if</tt> block; e.g., the following does not give an error:
+      <pre>
+      #if (true) { group g { is nonexisting_template; }}
+      </pre>
+      We have added a new compiler flag
+      <tt>--no-compat=broken_conditional_is</tt> that enables this error;
+      this flag will be automatically enabled in Simics 8
+      <bug number="SIMICS-22403"/>.</add-note></build-id>
```

Comment:
This releasenote feels too informal. I'd formulate it closer to how I did with `shared_logs_on_device`:
```
Adressed a bug where DMLC would not reject the instantiation of a non-existent template within an <tt>if</tt> block.
This bugfix is only applied